src/components/uri/Visualization.tsx

```diff
@@ -345,6 +349,13 @@ export class VisualizationWrapped extends React.Component<
                 // we do not need to pass totals={totals} because BucketPivotTable deals with changes in totals itself
                 return <PivotTableComponent {...commonProps} {...pivotBucketProps} />;
             }
+            case VisualizationTypes.PUSHPIN:
+                // TODO: will be fixed in SD-759
+                const geoBucketProps = {
+                    // ...mdObject,
+                    // content: fillMissingTitles(mdObject.content, locale),
+                };
+                return <GeoChartComponent {...commonProps} {...geoBucketProps} />;
```

Comment:
I think we should archive this file and implement it fully in [SD-759](https://jira.intgdc.com/browse/SD-759)